lib/pelion/Kconfig

```diff
@@ -0,0 +1,146 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# The purpose of this file is to override and extend settings related to
+# building Pelion cloud client library.
+
+config MBEDTLS_PROMPTLESS
+	bool
+	default y if !CUSTOM_PELION_SECURITY
+
+config NORDIC_SECURITY_PROMPTLESS
+	bool
+	default y if !CUSTOM_PELION_SECURITY
+
```

Comment:
Should be deleted, they have been moved to line L24-L31 to be inside the `PELION_SOURCES` safeguard.
```suggestion
```

Reply:
Removed